shared/wallets/common/account-reloader.tsx

```diff
@@ -29,7 +29,10 @@ const AccountReloader = (props: Props) => (
 const mapStateToProps = () => ({})
 
 const mapDispatchToProps = dispatch => ({
-  onReload: () => dispatch(WalletsGen.createLoadAccounts({reason: 'initial-load'})),
+  onReload: () => {
+    dispatch(WalletsGen.createLoadAccounts({reason: 'initial-load'}))
+    dispatch(WalletsGen.createUpdateAirdropDetails())
```

Comment:
Is this gonna spam the server too much?

Reply:
@songgao updateAirdropDetails is just getting a static JSON blob from the server's config, each time you visit the wallet tab.